javascript/ql/test/library-tests/RegExp/Bounds/tst.js

```diff
@@ -0,0 +1,3 @@
+/a{1}/;
+/a{1,}/;
+/a{1,5}/;
```

Comment:
Can we add the non-bound `/a{,1}/;` for completeness?

(`/a{,1}/.test('a{,1}') === true`)

Reply:
That's a syntax error, I believe.

Reply:
But I may be wrong, let me check...

Reply:
And indeed I'm wrong, we are more lenient than the ECMAScript standard. Yay, us!

Reply:
Apparently we treat a missing lower bound as zero, which is not how v8 handles it. That is, however, orthogonal to this PR.